### PR TITLE
React calculator

### PR DESCRIPTION
**Pull request by NupharP123**

### CODE REVIEW

File: React/CalcApp/e2e/project.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (OrMenashe) [inline React/CalcApp/e2e/project.json]: is this necessary? we will not deal with e2e so i do not want to create confusion

File: React/CalcApp/e2e/src/e2e/app.cy.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { getGreeting } from '../support/app.po';
+
+describe('calc-app', () => {
```
Comment (OrMenashe) [inline React/CalcApp/e2e/src/e2e/app.cy.ts]: is this necessary? we will not deal with e2e so i do not want to create confusion

File: React/CalcApp/e2e/src/fixtures/example.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "name": "Using fixtures to represent data",
```
Comment (OrMenashe) [inline React/CalcApp/e2e/src/fixtures/example.json]: is this necessary? we will not deal with e2e so i do not want to create confusion

File: React/CalcApp/package.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    "build": "nx build",
+    "test": "nx test"
+  },
+  "private": true,
+  "dependencies": {
+    "@swc/helpers": "~0.5.0",
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
+  },
+  "devDependencies": {
```
Comment (OrMenashe) [inline React/CalcApp/package.json:16]: can you please add jest and react testing library to the project?

File: React/CalcApp/e2e/project.json (hunk 1 above)
Comment (NupharP123) [reply React/CalcApp/e2e/project.json]: I removed the e2e folder with all its content

File: React/CalcApp/e2e/src/e2e/app.cy.ts (hunk 2 above)
Comment (NupharP123) [reply React/CalcApp/e2e/src/e2e/app.cy.ts]: I removed the e2e folder with all its content

File: React/CalcApp/e2e/src/fixtures/example.json (hunk 3 above)
Comment (NupharP123) [reply React/CalcApp/e2e/src/fixtures/example.json]: I removed the e2e folder with all its content

File: React/CalcApp/package.json (hunk 4 above)
Comment (NupharP123) [reply React/CalcApp/package.json:16]: react testing library was already there. I added jest

**Review by OrMenashe — APPROVED** [review]

**Pull request merged**